public/video-ui/styles/abstracts/_variables.scss

```diff
@@ -78,8 +78,8 @@ $darkGreyBorderColor: $color650Grey;
 /*
 Text colours
 */
-$brandColor: #297f5d;
-$darkBrandColor: #21674b;
+$brandColor: #ffbc01;
+$darkBrandColor: darken($brandColor, 2%);
```

Comment:
No need for % here, the unit is a percentage anyway I believe.
Also, is 2% _really_ all that much of a change?